src/sparsezoo/v2/test.md

```diff
@@ -0,0 +1,100 @@
+# API User pathway
+
+## Download one file
+
+### Through the string arguments
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "recipe=transfer_learn"
+model = Model(path=stub)
+model.download() # will only download only recipe file (as indicated by the string argument). If no argument provided, .download() will download to cache
+recipes: str = model.recipes.downloaded_path() 
+...
+```
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "deployment" # if we want the whole directory, the parameter does not have to have value
+model = Model(path=stub)
+model.download()
+deployment: str = model.recipes.downloaded_path() 
+...
+```
+### Through the API purely
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+model = Model(path=stub)
+recipe: str = model.recipes.download(recipe="transfer_learn")
+```
+
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+model = Model(path=stub)
+recipe: str = model.deployment.download()
+```
+
+## Download multiple chosen files (also in parallel)
+
+### Through the string arguments
```

Comment:
similar to above, the request for which files to be downloaded should be done through the python api, the query args should just be for filtering out specific kinds of files

ie

```
stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
# choose the files through the string arguments
stub = stub + "?" + "recipe=transfer_learn" + "&" + "checkpoint=preqat" + "&" + "recipe=original"

model = Model(stub)
model.download()
```
would download specified files + the sample inputs/outputs etc
and 
```python
model.training.downloaded_path()
```
for the same model would just download the preqat training checkpoint

Reply:
got it